CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/delete/DeleteSelectionPlugin.java

```diff
@@ -34,6 +36,7 @@
  */
 @ServiceProvider(service = Plugin.class)
 @Messages("DeleteSelectionPlugin=Delete Selection")
+@PluginInfo(pluginType = PluginType.DELETE, tags = {"DELETE", "MODIFY"})
```

Comment:
I think I would remove the modify tag here as I think it makes sense to distinguish between creating, modifying, and deleting